youtube.js

```diff
@@ -179,10 +179,13 @@ $("body").unbind("click").on("click", "yt-live-chat-text-message-renderer,yt-liv
 });
 
 function hideActiveChat() {
+  var html = '<div class="hl-c-cont fadeout"></div>';
```

Comment:
This line appears to be unnecessary, right?

Reply:
Yes, I forgot to remove it after trying "" instead of the div.